media/js/firefox/hello/index.js

```diff
@@ -127,6 +136,13 @@
                     $('.try-hello').on('click', function(e) {
                         e.preventDefault();
 
+                        // (bug 1115227) pass source to FTU
+                        if (tourSource === 'none') {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_productPage');
+                        } else {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_' + tourSource);
```

Comment:
Registering the pageID for the `none` case was superfluous as there is nothing to track. I will remove it.

We're now only registering a page ID if `utm_content` is populated and the Hello panel is opened. I don't think it make sense for people sharing the URL to have to enter some kind of special prefix to attribute their channel work in the FTU.